lib/kafka-consumer.js

```diff
@@ -124,7 +125,8 @@ function KafkaConsumer(conf, topicConf) {
   this.globalConfig = conf;
   this.topicConfig = topicConf;
 
-  this._consumeTimeout = 1000;
+  this._consumeTimeout = DEFAULT_CONSUME_LOOP_TIMEOUT_DELAY;
```

Comment:
Here you should use `DEFAULT_CONSUME_TIME_OUT` instead of `DEFAULT_CONSUME_LOOP_TIMEOUT_DELAY`